apis/Google.Cloud.Metadata.V1/Google.Cloud.Metadata.V1.IntegrationTests/Emulator/metadata_server_test.py

```diff
@@ -0,0 +1,456 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import OrderedDict
+import common, datetime, json, os, urllib2
+
+# These tests can be run with pytest: http://doc.pytest.org/en/latest/index.html
+
+try:
+  server_address = 'http://' + os.environ[common.host_environment_variable]
+  request = urllib2.Request(server_address,
+                            headers={common.metadata_flavor : common.metadata_flavor_google})
+  urllib2.urlopen(request)
```

Comment:
Ok, I moved this check to the places where we actually need to make network requests. We will only check that the environment variable is set here.